### ISSUE TITLE
linhpsdr creates FFTW3 wisdom file very time it is started

### ISSUE THREAD

**Issue by g0hww** [g0orx/linhpsdr#9]
On my Ubuntu 16.04LTS x96_64 box, linhpsdr claims that it is creating the FFTW3 wisdom file every time it starts.
I get the impression that the wisdom file should end up in the ~/.local/share/linhpsdr directory, which does exist, but there are no files in there.

**Comment by g0orx** [comment]
That sounds like a permissions problem.  Did you do "sudo make install"?  The README.md states to use "make install".

**Comment by g0hww** [comment]
Yes. I did "sudo make install" and had been launching linhpsdr via its icon without elevated privileges.
However, I did spot a post on a mailing list which suggested that running linhpsdr as root worked around this issue. I then ran "sudo linhpsdr" and let it crunch the wisdom, then exited. I then used chown to chnge the ownership of the content of .local/share/linhpsdr to my normal user account and relaunched linhpsdr.  Now it no longer crunches the wisdom every time it starts. Furthermore, bookmarks seem to work now, I had no success creating bookmarks before this, but that could have just been finger trouble.

**Comment by g0orx** [comment]
You should not have done "sudo make install" that was the initial problem.
